stable/nginx-ingress/README.md

```diff
@@ -51,6 +51,8 @@ Parameter | Description | Default
 `controller.image.pullPolicy` | controller container image pull policy | `IfNotPresent`
 `controller.config` | nginx ConfigMap entries | none
 `controller.defaultBackendService` | default 404 backend service; required only if `defaultBackend.enabled = false` | `""`
+`controller.scope.enabled` | limit the scope of the ingress controller | `false` (watch all namespaces)
```

Comment:
My goal was to have `all` as the default (same behavior as current master)  and add the option to set it to `{{ .Release.Namespace}}`. Maybe you can point me to a better solution, but I have not figured out a simpler way to do that.